assets/js/googlesitekit/widgets/datastore/constants.js

```diff
@@ -20,3 +20,6 @@ export const WIDGET_STYLES = { BOXES: 'boxes', COMPOSITE: 'composite' };
 export const WIDGET_WIDTHS = { QUARTER: 'quarter', HALF: 'half', FULL: 'full' };
 
 export const STORE_NAME = 'core/widgets';
+
+export const CONTEXT_DASHBOARD = 'dashboard';
+export const CONTEXT_PAGE_DASHBOARD = 'pageDashboard';
```

Comment:
See my other comment, these constants should probably go into a file outside of the datastore.

It could also be worth defining constants for all widget areas themselves and exposing them on the `Widgets` object so that 3P developers can rely on them when registering their own widgets.

Reply:
Ok, updated. I have also decided to rename area slugs not to contain context prefix (`AllTraffic` instead of `dashboardAllTraffic` and `pageDashboardAllTraffic`). It let's us easier define areas and assign it to multiple contexts. Does it work for you?

Reply:
We had discussed that a while ago, so the widget areas should keep these long names because they should be unique. I can see why that might seem odd, but they should use the fully qualified names. When registering widgets, you only specify the widget area they should be added to, so therefore the widget areas cannot just be e.g. `allTraffic` since then that widget would be added to the widget area in both contexts, which is not what you might want.

Reply:
Okay. Updated.